packages/elements-core/src/components/Docs/Model/Model.tsx

```diff
@@ -6,14 +7,22 @@ import * as React from 'react';
 
 import { SchemaAndDescription } from '../../SchemaAndDescription';
 import { DocsComponentProps } from '..';
+import { InternalBadge } from '../HttpOperation/Badges';
 
 export type ModelProps = DocsComponentProps<JSONSchema7>;
 
 const ModelComponent: React.FC<ModelProps> = ({ data, className, headless }) => {
+  const isInternal = !!data['x-internal'];
```

Comment:
So models have `x-internal` property and operations have `internal`? 🤔 

Wouldn't it be better to unify that in `http-spec`?

Reply:
Ah, I've seen other comments. @philsturgeon is this okay? 🤔

Reply:
I’m ok with it if you folks are. Doesn’t effect product so long as it works.

Reply:
I would of course prefer to see it unified.

But if for some reason that is difficult/impossible, I'll accept the PR as is.

Reply:
If it's not a blocker and is something that can happen separately from this PR, I can make an issue for it and address it here as a follow up item. But if it's non-trivial, I'll take care of that first.